README.md

```diff
@@ -1,17 +1,13 @@
 # RailsAdmin
 
 [![Gem Version](https://img.shields.io/gem/v/rails_admin.svg)][gem]
-[![Build Status](https://img.shields.io/github/workflow/status/sferik/rails_admin/Test)][ghactions]
-[![Coverage Status](https://img.shields.io/coveralls/sferik/rails_admin.svg)][coveralls]
-[![Inline docs](http://inch-ci.org/github/sferik/rails_admin.svg)][inch]
-[![Code Climate](https://codeclimate.com/github/sferik/rails_admin.svg)][codeclimate]
```

Comment:
I have set up Code Climate for the new repo, please add it back.
https://codeclimate.com/github/railsadminteam/rails_admin

Reply:
Done. Restored. Sorry it took me so long to respond.